packages/forms-web-app/src/pages/projects/section-51/index/view.njk

```diff
@@ -27,48 +27,49 @@
 				</span>
 
 				<h1 class="govuk-heading-xl">
-					{{ title }}
+					{{ t('section51Index.heading1') }}
 				</h1>
 
-				{% if advice.length > 0 or(advice.length == 0 and queryUrl != '') %}
-					<p class="govuk-body">The list below includes a record of advice we have provided for this project.</p>
+				{% if advice.length > 0 or (advice.length == 0 and (queryUrl != '?lang=en' and queryUrl != '?lang=cy' and queryUrl != '')) %}
```

Comment:
This is not very readable. Maybe it's worth having something like `adviceExists` and `showNoAdviceMwssage` etc returned from the controller/view-model to deal with the conditional rendering of those elements?